conans/client/tools/system_pm.py

```diff
@@ -114,6 +116,18 @@ def install(self, packages, update=True, force=False):
             self.update()
         self._install_any(packages)
 
+    def _parse_packages_arch(self, packages, platforms):
+        if self._settings and cross_building(self._settings):
+            _, build_arch, _, host_arch = get_cross_building_settings(self._settings)
+            arch = host_arch or build_arch
+            parsed_packages = []
+            for package in packages:
+                if isinstance(package, str):
```

Comment:
what do we have besides strings?

Reply:
line 94: `packages = [packages] if isinstance(packages, str) else list(packages)`
but I see your point, it doesn't make sense. I'll remove it.

Reply:
done